docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-15.rst

```diff
@@ -0,0 +1,174 @@
+Squeeze
+=======
+
+
+.. meta::
+  :description: Learn about Squeeze-15 - a shape manipulation operation, which
+                can be performed on one required and one optional input tensor.
+
+**Versioned name**: *Squeeze-15*
+
+**Category**: *Shape manipulation*
+
+**Short description**: *Squeeze* removes dimensions equal to 1 from the first input tensor.
+
+**Detailed description**: *Squeeze* can be used with or without the second input tensor.
+
+* If only the first input is provided, every dimension that is equal to 1 will be removed from it.
+* With the second input provided, each value is an index of a dimension from the first tensor that is to be removed. Specified dimension should be equal to 1, otherwise it will be ignored and copied as is.
+  Dimension indices can be specified directly, or by negative indices (counting dimensions from the end).
+
+.. note::
+
+    - If index of the dimension to squeeze is provided as a constant input and it points to a dynamic dimension that might be `1`, then the dimension is considered as squeezable. Therefore the rank of the output shape will be reduced, but not dynamic.
+    - If the input with indices is empty or not provided, dynamic dimension compatible with `1` leads to dynamic rank of the output shape.
+
+
+**Attributes**:
+
+* *allow_axis_skip*
+
+  * **Description**: Shape inference result dynamic rank if selected axis has value 1 in range of its dynamic.
```

Comment:
Just a suggestion, it's up to whether to apply it.
```suggestion
  * **Description**: If true, shape inference results in a dynamic rank if selected axis has value 1 in its dynamic range.
```

Reply:
Done